### DISCUSSION TITLE
launching it

### DISCUSSION THREAD

**Discussion by demonshe11** [titaniumnetwork-dev/Vanadium#12] [Q&A] [answered]
how are you supposed to launch it through repl.it?

**Comment by Null-n-V0id** [comment]
If repl.it continues to not function, it's recommended to use Heroku (deploy button), or https://glitch.me for hosting this website.

**Comment by pbkx** [comment, marked as answer]
@demonshe11 
If your still wondering, you can click the deploy button in readme.
In theory, it should work automatically.
However, if it doesn't launch, in the .replit file try changing "npm start" to "npm install"
